build/deploy/main.go

```diff
@@ -83,15 +83,15 @@ func uploadPlugin(client *model.Client4, pluginID, bundlePath string) error {
 	defer pluginBundle.Close()
 
 	log.Print("Uploading plugin via API.")
-	_, resp := client.UploadPluginForced(pluginBundle)
-	if resp.Error != nil {
-		return fmt.Errorf("Failed to upload plugin bundle: %s", resp.Error.Error())
+	_, _, err = client.UploadPluginForced(pluginBundle)
+	if err != nil {
+		return fmt.Errorf("Failed to upload plugin bundle: %s", err.Error())
```

Comment:
nit: `errors.Wrapf()` here would be just a bit cleaner.

Reply:
ah yeah, you can see how I was just kind of doing a dumb translation to correct the number of return values and argument types and didn't really think about what was actually being called to create the error, here